src/components/UnderlyingToken.tsx

```diff
@@ -43,19 +45,22 @@ export const UnderlyingToken = ({ assetId }: UnderlyingTokenProps) => {
     ;(async () => {
       try {
         if (shouldHide || !wallet) return
+        moduleLogger.trace({ asset, fn: 'yearn.token' }, 'Get Yearn Token')
```

Comment:
I'm not sure what syntax we want to stick with...

```suggestion
        moduleLogger.trace({ asset, fn: 'yearn.token()' }, 'Get Yearn Token')
```

Reply:
Probably without the `()`, as it isn't used elsewhere?

I do like it though, makes it clear that it's a function.

Reply:
I mean... it's named `fn`.